R/parse-block.R

```diff
@@ -273,6 +287,9 @@ evaluateBlock <- function(srcref, file, expr, envir, addEndpoint, addFilter, mou
     filter <- PlumberFilter$new(block$filter, expr, envir, block$serializer, srcref)
     addFilter(filter)
 
+  } else if (!is.null(block$errorhandler)){
+    setErrorHandler(eval(expr))
```

Comment:
Since the plumberRouter is now passed to `evaluateBlock`, use pr$setErrorHandler directly